M2/Macaulay2/packages/JSON.m2

```diff
@@ -358,8 +358,5 @@ for tst in sort select(tsts, f -> match("^y_", f)) do (
 close outfile
 ///
 
-TEST(currentPackage#"source directory" | "JSON/test-parse.m2",
-    FileName => true)
-
-TEST(currentPackage#"source directory" | "JSON/test-encode.m2",
-    FileName => true)
+TEST get(currentPackage#"source directory" | "JSON/test-parse.m2")
```

Comment:
I _think_ just `TEST "./JSON/..."` should always work, no?

Reply:
It doesn't :(  I get the following:

```m2
i1 : check "JSON"
../src/macaulay2/M2/M2/Macaulay2/packages/JSON.m2:361:0:(2):[18]: error: opening input file "./JSON/test-parse.m2" failed: No such file or directory
```

I think it depends on the value of `currentDirectory()`.  If I `changeDirectory` into the packages directory, then it works just fine.

Reply:
Hmm but `load "./..." works correctly. What's the difference Does `TEST(currentFileDirectory | ...)` work?

Reply:
Yes, it works with `currentFileDirectory`

Reply:
Ah, the difference is that `load` looks everywhere under `path` but `get` doesn't. A little strange to be honest.